resources/views/projects/show-language-selection.blade.php

```diff
@@ -7,14 +7,14 @@
         </ol>
         <p class="h3">{{ __('Create new project') }}</p>
         <h1>
-            {{ __('Project Translations') }}
```

Comment:
I'm not sure this should have been changed. See: https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A17897

@cherylhjli Can you clarify?